src/e2e/java/teammates/e2e/pageobjects/AdminSearchPage.java

```diff
@@ -210,7 +204,7 @@ public void resetInstructorGoogleId(InstructorAttributes instructor) {
         instructorRow.findElement(By.linkText(LINK_TEXT_RESET_GOOGLE_ID)).click();
         waitForPageToLoad();
 
-        resetGoogleIdModal.findElement(By.className("btn-warning")).click();
+        waitForConfirmationModalAndClickOk();
```

Comment:
Possible to split this method into two? so `waitForConfirmationModal()` and `clickOkOnModal()`

Reply:
What's the benefit of doing that? There's a `waitForModalShown()` already, and almost all modal actions are doable by `waitForConfirmationModalAndClickOk` and `waitForConfirmationModalAndClickCancel()` anyway.

Reply:
Yeah, just thought we could be clearer by separating the steps up further. But agree that modal actions are simple enough so let's leave it